package.json

```diff
@@ -17,6 +17,7 @@
     "@bcrs-shared-components/corp-type-module": "1.0.13",
     "@bcrs-shared-components/enums": "1.0.44",
     "@bcrs-shared-components/breadcrumb": "2.1.24",
+    "@bcrs-shared-components/business-lookup": "1.1.29",
```

Comment:
Please do a `npm i` and also commit the package-lock file.

Reply:
I did